distribution/pom.xml

```diff
@@ -33,10 +33,15 @@
         <karaf.version>2.4.1</karaf.version>
         <pax.web.version>3.1.2</pax.web.version>
         <pax.cdi.version>0.8.0</pax.cdi.version>
+        <ddf.platform.app.version>2.8.0-SNAPSHOT</ddf.platform.app.version>
+        <ddf.content.app.version>2.8.0-SNAPSHOT</ddf.content.app.version>
+        <ddf.catalog.app.version>2.8.0-SNAPSHOT</ddf.catalog.app.version>
+        <ddf.searchui.app.version>2.8.0-SNAPSHOT</ddf.searchui.app.version>
+        <ddf.spatial.app.version>2.8.0-SNAPSHOT</ddf.spatial.app.version>
+        <ddf.admin.app.version>1.3.0-SNAPSHOT</ddf.admin.app.version>
```

Comment:
Since Admin got merged into Platform this now should be Platform's version (currently 2.8.0-SNAPSHOT). Otherwise, some of the recent admin doc updates will not be picked up.